go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/mr-tron/base58 v1.2.0
 	github.com/nspcc-dev/neo-go v0.106.3
 	github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20240729160116-d8e3e57f88f2
+	github.com/nspcc-dev/neofs-sdk-go v1.0.0-rc.12
```

Comment:
I'm not excited about this dependency, can we avoid it?

Reply:
sure, only tests use it